index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <link href='./style.css' rel='stylesheet'>
-        <title>Survey Form</title>
-    </head>
+     <head>
+         <link href='./style.css' rel='stylesheet'>
+         <title>Survey Form</title>
+     </head>
     <body>
         <h1 id='title'>freeCodeCamp Survey Form</h1>
         <p id='h1'>Thank you for taking time to help us improve the platform</p>
```

Comment:
Hi @HAMMAS-SALEEM, When making reviews, you have to be more specific on where to use a semantic tag and why is best to use a semantic tag. You said for paragraph and heading use semantic tag, why?

Reply:
should I remove all my comments and recreate them.

Reply:
Could you suggest me some kind of video and one more thing do i need to use inline comments or not because i don't understand in your comment what you said about inline and i didn't setup linters then. So it is a comment which i gave as an change.